datafusion/common/src/dfschema.rs

```diff
@@ -619,75 +779,77 @@ impl DFSchema {
     /// Strip all field qualifier in schema
     pub fn strip_qualifiers(self) -> Self {
         DFSchema {
-            fields: self
-                .fields
-                .into_iter()
-                .map(|f| f.strip_qualifier())
-                .collect(),
-            ..self
+            inner: self.inner.clone(),
+            field_qualifiers: vec![None; self.inner.fields.len()],
+            functional_dependencies: self.functional_dependencies.clone(),
```

Comment:
I think the previous syntax avoid clone.

Reply:
done